src/communities/searchSummary.ts

```diff
@@ -0,0 +1,59 @@
+import { camelizeKeys } from "humps"
+import { authenticatedRequest } from "../requests"
+import { token } from "../token"
+import {
+  ICommunity,
+  ICredentials,
+  IEnvironment,
+  IFetchExecutor,
+  IMHBOListing,
+  ISearchParams,
+  IUnparsedMHBOListing
+} from "../types"
+import queryBuilder from "./queryBuilder"
+
+/**
+ * Performs a search for mobile homes.
+ *
+ * @param creds The authentication credential for the API request.
+ * @param environment An optional override of the environment to utilize.
+ * @param fetchExecutor An instance of the request executor.
+ * @returns An array of mobile home results.
+ */
+async function searchSummary(
+  params: ISearchParams,
+  creds: ICredentials,
+  environment?: IEnvironment,
+  fetchExecutor?: IFetchExecutor
+): Promise<ICommunity[]> {
+  params.detailLevel = "SUMMARY"
+  const response = await authenticatedRequest(
```

Comment:
Since the code for search and summary are only a line different, can we call the other method or compose a new method from a shared module?

import search from './search'
async function searchSummary(
.....
return search(params, creds,environment,fetchExecutor)
}

Reply:
directly calling the other method may not be possible due to typecasting issue, I'll refactor into a new method